### PR TITLE
Make the update retrigger instructions clear

### LINKED ISSUE #829: UX: make the retrigger instructions clear
As we can see [here](https://github.com/containerbuildsystem/dockerfile-parse/issues/104), the instructions `You can re-trigger the update by adding /packit propose-update to the issue comment.` can be understood as that you need to edit the original comment (=issue description) instead of adding a new comment.

AC:
- [ ] Unify the sentence with other occurences of retrigger wording.
- [ ] Make it clear that users need to add a new comment.

### PR DESCRIPTION
**Pull request by lbarcziova**
To have it consistent with the one for build:
https://github.com/packit/packit-service/blob/45903cf592dd7be8bff474c8c81b25e1a9dde8f6/packit_service/constants.py#L21-L24
Fixes #829

### CODE REVIEW

File: packit_service/worker/handlers/github_handlers.py [hunk 1]
```diff
@@ -211,8 +211,8 @@ def run(self) -> TaskResults:
                 f"| dist-git branch | error |\n"
                 f"| --------------- | ----- |\n"
                 f"{branch_errors}\n\n"
-                "You can re-trigger the update by adding `/packit propose-update`"
-                " to the issue comment.\n"
+                "You can re-trigger the update by adding a comment (`/packit propose-update`)"
```
Comment (mfocko) [inline packit_service/worker/handlers/github_handlers.py:214]: could you please change to `retrigger`? #grammarPolice

**Review by mfocko — COMMENTED** [review]
Since you have the snippet in the PR description, could it be used as format string? Appears to differ only in the command to be used and build vs update.

Otherwise LGTM :)

**Review by TomasTomecek — COMMENTED** [review]
Nice wording; @mfocko +1, [retrigger](https://www.lexico.com/definition/retrigger) is the correct word (even though e.g. firefox's spellcheck doesn't like it)

File: packit_service/worker/handlers/github_handlers.py (hunk 1 above)
Comment (lbarcziova) [reply packit_service/worker/handlers/github_handlers.py]: changed

**Review by lachmanfrantisek — APPROVED** [review]
Thanks, nice improvement!

:+1: from me

**Review by mfocko — APPROVED** [review]
:+1:

**Pull request merged**